sdks/go/pkg/beam/core/graph/coder/coder.go

```diff
@@ -126,8 +126,17 @@ const (
 	VarInt        Kind = "varint"
 	WindowedValue Kind = "W"
 	KV            Kind = "KV"
-	GBK           Kind = "GBK"
-	CoGBK         Kind = "CoGBK"
+
+	// CoGBK is non-standard and currently equivalent to either
```

Comment:
<!--thread_id:cc_166096618_t; commit:977a4f133a10d957b90566426581193d93041321; resolved:1-->
<!--section:context-quote-->
> **wcn3** wrote:
> I have some mild aversion to the phrase "non-standard" when the "standard" in question isn't obvious. Let's just say this the same way it's said elsewhere in the PR: CoGBK is not (yet) an official Beam primitive (JIRA pending).

<!--section:body-->
Removed the phrase. The coder part is less clear cut, given that the coder for the result of CoGBK doesn't even have a proposed encoding equivalent to KV<K,Stream<V>> yet.